pkg/github/github.go

```diff
@@ -128,6 +132,15 @@ type Client interface {
 	) (*github.IssueComment, *github.Response, error)
 }
 
+// NewIssueOptions is a struct of optional fields for new issues
+type NewIssueOptions struct {
+	Assignee  string   // GitHub handle of main assignee
```

Comment:
Why do we need to distinguish between the main assignee and other assignees? Is that coming from the GitHub API?

Reply:
Yes it comes from the GitHub API. But you gave me an idea, we can simplify our options struct more and assign them automatically from a single slice of usernames :+1: